Jenkinsfile

```diff
@@ -117,35 +113,47 @@ pipeline {
             steps {
                 script {
                     lock(label: 'testenv_pool', quantity: 1) {
-                        testNetwork = getLockedNetworkName()
-                        vmwareConfig = getVMwareConfig()
-                        testEnvName = getTestEnvName(testNetwork)
-                        testEnvFolder = env.VC_FOLDER
+                        // 'Cleanup' stage
+                        node(label: 'ansible') {
```

Comment:
This comment is unrelated to this block.

Also, this code could be run outside the `node` block. (if not, please explain why it needs to be run inside `node`)

Reply:
It needs to be in `node`, because script uses `env.WORKSPACE` which is only defined when job is running on a given node.

But yes, comment can be deleted.